docs/compass/04-01-compass.md

```diff
@@ -3,14 +3,23 @@ title: Enable Compass in Kyma
 type: Installation
```

Comment:
file' names of those document is a mistake. We have clearly compass-installation-options and it's called 04-01-compass, why?!

Reply:
I'm afraid that there is no other way but to discuss this issue at the SIG Content.